app/views/layouts/listnav/_container_group.html.haml

```diff
@@ -36,3 +36,4 @@
         = multiple_relationship_link(@record, :container)
         = multiple_relationship_link(@record, :container_image)
         = single_relationship_link(@record, :container_node)
+        = multiple_relationship_link(@record, :persistent_volumes)
```

Comment:
`:persisent_volume` .. this should make `multiple_relationship_link` use the right rbac feature, while still accessing the right data (as it does call `table.pluralize` for that).